package.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "vue-adjmatrix",
+  "name": "MultiMatrix",
```

This name will be what NPM uses to publish this as a package. If we were to ever publish this, we'd want to use the lowercase name (same as the GitHub repo).

```suggestion
  "name": "multimatrix",
```